### ISSUE TITLE
XmlIgnoreAttribute (and other types) exists in both System.Xml.Serialization and netstandard

### ISSUE THREAD

**Issue by lemoinem** [reinforced/Reinforced.Typings#125]
Hi,

I have a netstandard2.0 library using Reinforced.Typings 1.5.2 (from NuGet).
However, the library also uses the `[Xml*]` attributes.

As soon as these attributes are used, I'm getting the following error:
The type 'XmlIgnoreAttribute' exists in both 'System.Xml.XmlSerializer, Version=4.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' and 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'

Reproduction step:

1. Net netstandard2.0 library project
2. Add Reinforced.Typings 1.5.2 with Nuget
3. Create following class:
```
namespace ClassLibrary1
{
	using System.Xml.Serialization;

	public class Class1
	{
		[XmlIgnoreAttribute]
		public int MyInt { get; set; }
	}
}
```
4. Build

I'm not what needs to be fixed or if it's possible... Anyone has an idea?
This is a blocker for us as we need it to migrate a library to .Net Core...

**Comment by pavel-b-novikov** [comment]
This is RT build issue. Will be fixed today

**Comment by pavel-b-novikov** [comment]
Fixed in version 1.5.3
